admin-dev/themes/new-theme/js/pages/module/controller.js

```diff
@@ -224,11 +224,19 @@ class AdminModuleController {
 
   initBOEventRegistering() {
     window.BOEvent.on('Module Disabled', this.onModuleDisabled, this);
+    window.BOEvent.on('Module Uninstalled', this.updateTotalResults, this);
   }
 
   onModuleDisabled() {
-    this.getModuleItemSelector();
-    // Don't care nothing to do?
+    const self = this;
+    const moduleItemSelector = self.getModuleItemSelector();
+
+    $('.modules-list').each(function scanModulesList() {
+      self.updateTotalResults(
```

Comment:
Is it me or this method doesn't accept arguments?

Reply:
True, it's a function from the old js =)

Reply:
Done!